component/src/main/wso2/sap/Module.md

```diff
@@ -110,36 +110,36 @@ public function main() {
 ```ballerina
 
 listener sap:Listener consumerEP = new ({
-    transportName:"<The protocol name[idoc/bapi]>",
-    serverName:"<Name of the server configuration>",
-    gwhost:"<Gateway host on which the server should be registered>",
-    progid:"<The program ID with which the registration is done>",
-    repositorydestination:"<Name of the repository>",
-    gwserv:"<Gateway service>",
-    unicode:"<Determines whether or not you connect in unicodemode>"}, 
+    transportName: "<The protocol name[idoc/bapi]>",
+    serverName: "<Name of the server configuration>",
+    gwhost: "<Gateway host on which the server should be registered>",
```

Comment:
Again missing some camel cases here. Please check all the lines.

Reply:
As per the Rajkumar, customer wanted this notation for sap parameters.